docs/source/i18n/ultimate.es.rst

```diff
@@ -19,10 +19,30 @@ Si cambia de opinión y retira su consentimiento para participar en este estudio
+Desde nuestro conocimiento y mejor entender, los datos que compartimos con el público general no contendrán información que pueda identificarlo directamente. Los datos que compartimos no contendrán su nombre, solo un código numérico, por lo que la gente no sabrá su nombre o qué datos son suyos. Además, los datos puestos a disposición del público en general no incluirán datos que pensamos que podrían ayudar a las personas que lo conocen a adivinar qué datos son suyos, como sus rasgos faciales o la fecha en que participó en el estudio. Si escribimos un informe o artículo sobre este estudio o compartimos el conjunto de datos del estudio con otras personas, lo haremos de tal manera que usted no pueda ser identificado directamente. Sin embargo, al utilizar datos adicionales vinculados a su nombre (por ejemplo, escáneres cerebrales obtenidos de sus registros médicos), uno podría potencialmente relacionar sus imágenes u otra información en nuestra base de datos con usted. Además, una brecha de seguridad (intrusión o ciberataque) podría hacer que alguien pudiera asociarlo a sus datos. Este riesgo es muy bajo porque sus datos se almacenan en una base de datos segura y la información sobre su identidad se almacena de manera separada a los datos en sí, únicamente vinculados a través de un código.
 
 Mantendremos la parte privada de sus datos (nombre, información para contactarle, etc.) en un lugar seguro durante al menos <x> años. De esta manera, si uno de los investigadores que utilice los datos de este estudio encuentra algo en sus escaneos cerebrales que pudieran tener un valor diagnóstico, podremos contactar con usted. Después de este periodo de tiempo, destruiremos su información para proteger su privacidad.
 
-**Dejarnos usar y compartir sus datos es voluntario. Sin embargo, tiene que estar dispuesto a compartir sus datos de esta forma para poder participar en este estudio.**
+**Permitirnos usar y compartir sus datos es voluntario. Sin embargo, tiene que estar dispuesto a compartir sus datos de esta forma para poder participar en este estudio. Si no está dispuesto a compartir sus datos, no podrá participar en el estudio.**
 
-Si no está dispuesto, no podrá participar en el estudio. Al firmar abajo, está aceptando proporcionar sus datos a futuras investigaciones. De esta forma acepta que los datos podrán ser compartidos con otros investigadores de otras instituciones por todo el mundo. Los detalles, resultados, e implicaciones de estos estudios son desconocidos.
+Al firmar abajo, está aceptando proporcionar sus datos a futuras investigaciones. De esta forma acepta que los datos podrán ser compartidos con otros investigadores de otras instituciones por todo el mundo. Los detalles, resultados, e implicaciones de estos estudios son desconocidos.
+
+
+Two access types version (some data shared publicly, more data shared to approved researchers)
```

Comment:
I've seen versions in other languages don't translate this, but in case it's needed:

```suggestion
Two access types version (some data shared publicly, more data shared to approved researchers)
*Versión para dos tipos de acceso (algunos datos se comparten públicamente, y otros datos se comparten con investigadores aprobados)*
```

Reply:
are we all good to merge this? i cannot comment on spanish